src/leader_scheduler.rs

```diff
@@ -20,7 +20,8 @@ use std::sync::Arc;
 
 pub const TICKS_PER_BLOCK: u64 = 4;
```

Comment:
Where are these `TICKS_PER_BLOCK` and `BLOCKS_PER_SLOT` numbers coming from.  The leader scheduler deals with ticks, slots and epochs not blocks

Reply:
yup! just updated it :D